resources/views/messages/reminder.blade.php

```diff
@@ -0,0 +1 @@
+<?php echo $content->body ?>
```

Comment:
@weerd This should definitely be blade, this is just a test to get things through. We eventually need more views, and the correct formatting for everything.